DOCUMENTATION.md

```diff
@@ -511,7 +515,7 @@ If the request to the client is successful, the client will return a `dict`:
   "line_4": "ADDRESS LINE 4", # optional string for letter
   "line_5": "ADDRESS LINE 5", # optional string for letter
   "line_6": "ADDRESS LINE 6", # optional string for letter
-  "postcode": "A REAL UK POSTCODE", # required string for letter
+  "line_7": "A REAL UK POSTCODE", # required string for letter
```

Comment:
do we need to add something about country name here?

Reply:
i'd probably just revert this to "ADDRESS LINE 7"